src/overlays/actors/ovl_En_Sw/z_en_sw.c

```diff
@@ -173,10 +173,10 @@ void func_808D8940(EnSw* this, PlayState* play) {
 }
 
 s32 func_808D8B58(EnSw* this) {
-    s16 phi_s2 = (s16)((s16)(Rand_ZeroOne() * 1000.0f) % ARRAY_COUNT(this->unk_464)) * 0x1555;
+    s16 phi_s2 = (s16)((s16)(Rand_ZeroOne() * 1000.0f) % ENSW_BODYPART_MAX) * 0x1555;
```

Comment:
Isn't this has the issue of casting a float to an s16?
```suggestion
    s16 phi_s2 = (s16)((s16)(s32)(Rand_ZeroOne() * 1000.0f) % ENSW_BODYPART_MAX) * 0x1555;
```
Not sure if we want to actually use the `TRUNCF_BINANG` macro here

Reply:
The `TRUNCF_BINANG` is fine here